client/src/modules/stock/exit/modals/findDepot.modal.html

```diff
@@ -27,15 +29,15 @@
 
     <bh-has-requisition-voucher
       message="$ctrl.requisitionMessage"
-      classe-label="$ctrl.requisitionLabel"
+      class-label="$ctrl.requisitionLabel"
       reference="$ctrl.reference"
       requestor="$ctrl.selected.uuid"
       on-change="$ctrl.onChangeReference(reference)">
     </bh-has-requisition-voucher>
 
     <bh-has-shipment-voucher
       message="$ctrl.requisitionMessage"
-      classe-label="$ctrl.requisitionLabel"
+      class-label="$ctrl.requisitionLabel"
```

Comment:
I agree with this change, but you'll need to change the component definitions as well to fix the `classe` typo.  Can you do that?

Reply:
Fixed